### DISCUSSION TITLE
Item Slot documentation doesn't mention get_item() function?

### DISCUSSION THREAD

**Discussion by OfficerBBB** [peter-kish/gloot#149] [Q&A] [answered]
Hi, very useful addon!

I'm working with ItemRefSlot and got confused because documentation didn't show a function for getting the InventoryItem held by the slot. I checked docs for ItemRefSlot, ItemSlot and ItemSlotBase but none of them had this function, and I wondered if I was misunderstanding how slots were meant to be used.

It was only when I checked the code that I found the **get_item()** function. Raising as this seems to be missing from docs?

Thanks!

**Comment by peter-kish** [comment, marked as answer]
Yup, you're right. I forgot to describe that function in the docs for `ItemSlotBase`, `ItemSlot` and `ItemRefSlot`.
Should be fixed now: 23c95b5aa3415488e3f1130ec96a96fe35efdf4f.

**Reply by OfficerBBB** [reply]
Great!
